### PR TITLE
fix(heartbeat-bar): cleanup css styling and minor syntax issues

### PR DESCRIPTION
**Pull request by bertyhell**
* Improved margins of the healthbar
* Minor eslint fixes for healthbar syntax

image

### CODE REVIEW

**Comment by louislam** [comment]
Padding left/right look great, but I prefer to keep the original padding top/bottom.

**Comment by bertyhell** [comment]
> Padding left/right look great, but I prefer to keep the original padding top/bottom.

I kept the top padding the same. You mean you want to increase the top padding to equal the bottom padding? Or do you want to keep the same height of the div (increase the top padding slightly and decrease the bottom padding)?

File: src/components/MonitorList.vue [hunk 1]
```diff
@@ -19,21 +19,21 @@
...
                             <Uptime :monitor="item" type="24" :pill="true" />
-                            {{ item.name }}
+                            <span class="ms-1">{{ item.name }}</span>
                         </div>
                         <div class="tags">
                             <Tag v-for="tag in item.tags" :key="tag" :item="tag" :size="'sm'" />
                         </div>
                     </div>
-                    <div v-show="$root.userHeartbeatBar == 'normal'" :key="$root.userHeartbeatBar" class="col-6 col-md-4">
+                    <div v-show="$root.userHeartbeatBar === 'normal'" :key="$root.userHeartbeatBar" class="col-6 col-md-4 small-padding">
```
Comment (louislam) [inline src/components/MonitorList.vue:31]: Maybe due to this? the padding top/left look small now.

Comment (bertyhell) [reply src/components/MonitorList.vue:31]: no, I made the top and bottom padding the same
image

and I removed this space below the health bar:
image


I'll change it so the total size of the monitor item stays the same

Comment (bertyhell) [reply src/components/MonitorList.vue:31]: fixed:
image

**Pull request merged**